kubecargoload.py

```diff
@@ -363,6 +375,14 @@ def _setup_options():
         help='list the requested object(s) across all namespaces',
         default=False)
 
+    argument_parser.add_argument(
+        '-c',
+        '--cpu',
+        dest='cpu',
```

Comment:
Maybe `show_cpu_usage` is a more descriptive name for the destination option name?
If you agree, I would suggest to also use it for the corresponding attribute on `KubernetesCargoLoadOverviewProvider` and `KubernetesCargoLoadOverviewPrinter`.

The command line option name seems fine to me though.

Reply:
Sounds good. Will make the changes.